frontend/src/pages/interview.js

```diff
@@ -0,0 +1,172 @@
+import Head from '../components/head'
+import Nav from '../components/nav'
+import Link from 'next/link'
+
+type Props = {}
+class Interview extends Component<Props> {
+  constructor(props) {
+    super(props)
+    this.state = { value: '' }
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+  handleSubmit(event) {
+    alert('A name was submitted: ' + this.state.value)
+    event.preventDefault()
+  }
+
+  render() {
+    return (
+      <Container>
+        <Head title="Interview" />
+        <Nav />
+        <Form>
+          <FormGroup>
+            <legend>Time</legend>
+            <FormGroup check>
+              <Label>
+                <Input type="checkbox" name="time" /> Exec member for another org
+              </Label>
+            </FormGroup>
+            <FormGroup check>
+              <Label>
+                <Input type="checkbox" name="time" /> Consulting club such as IBC, OTCR
+              </Label>
+            </FormGroup>
+            <FormGroup check>
+              <Label>
+                <Input type="checkbox" name="time" /> Some other club that requires time (ex:
+                Enactus, Fraternity Pledge during the same semester)
+              </Label>
+            </FormGroup>
+            <FormGroup check>
+              <Label>
+                <Input type="checkbox" name="time" /> Hard/Time-consuming classes
+              </Label>
+            </FormGroup>
+            <FormGroup check>
+              <Label>
+                <Input type="checkbox" name="time" /> Hard course-load (still take one point off for
+                each hard class they have as described above)
+              </Label>
+            </FormGroup>
+            <FormGroup check>
+              <Label>
+                <Input type="checkbox" name="time" /> Too many org obligations
+              </Label>
+            </FormGroup>
+          </FormGroup>
+          <FormGroup>
+            <legend>Initiative and Passion</legend>
+            <FormGroup check>
+              <Label>
+                <Input type="radio" name="passion" /> 0 - Definitely using this as a resume booster
+              </Label>
+            </FormGroup>
+            <FormGroup check>
+              <Label>
+                <Input type="radio" name="passion" /> 1 - Had a general idea of what hack4impact
+                was. They probably only read a couple sentences about Hack4Impact and applied and
+                that's it.
+              </Label>
+            </FormGroup>
+            <FormGroup check>
+              <Label>
+                <Input type="radio" name="passion" /> 2 - Really into it, thinking about ways they
+                could contribute to the org
+              </Label>
+            </FormGroup>
+          </FormGroup>
+          <FormGroup>
+            <legend>Community</legend>
+            <Label>
+              Will they contribute to community or are they just using this as a resume booster?{' '}
```

Comment:
whats the `{' '}` at the end?

Reply:
oh that's just a space for now ( it makes things more readable before we add css)